### PR TITLE
Refactoring previously tests and add testPipelineGroovyPageOpening  method test

### PR DESCRIPTION
**Pull request by RuslanG13**

### CODE REVIEW

**Comment by RuslanG13** [comment]
[AT_017.012 | Pipeline > Create Pipeline. Verify that the checkbox with "Use Groovy Sandbox" is checked by default when user setting up a Pipeline project](https://trello.com/c/qvgH58XN/290-at017012-pipeline-create-pipeline-verify-that-the-checkbox-with-use-groovy-sandbox-is-checked-by-default-when-user-setting-up-a)

File: src/test/java/RuslanGPipelineCreatePipelineTest.java [hunk 1]
```diff
@@ -10,30 +10,33 @@
...
                 .sendKeys("Ruslan Gudenko Pipeline Project+" + date.getTime());
         getDriver().findElement(By.xpath("//li[@class='org_jenkinsci_plugins_workflow_job_WorkflowJob']"))
                 .click();
-
-        WebElement okButton = getDriver().findElement(By.xpath("//button[@id='ok-button']"));
-        javascriptExecutor.executeScript("arguments[0].scrollIntoView();", okButton);
-        okButton.click();
-
+        getDriver().findElement(By.xpath("//button[@id='ok-button']"));
+        getDriver().findElement(By.xpath("//button[@id='ok-button']")).click();
```
Comment (azellus-tercia) [inline src/test/java/RuslanGPipelineCreatePipelineTest.java:26]: This line is the same as previous, except .click() added. I think you can remove previous line.

Comment (RuslanG13) [reply src/test/java/RuslanGPipelineCreatePipelineTest.java:25]: Yes. You're absolutely right. It seems to me that it was a mistake of copy-past. Deleted
 Thank you. You might approve it

**Review by korelov — APPROVED** [review]

**Review by azellus-tercia — APPROVED** [review]

**Review by VitalyVZH — APPROVED** [review]
Тест рабочий, но над упрощением локаторов стоит поработать

**Pull request merged**